docs/index.html

```diff
@@ -1404,8 +1404,8 @@ <h3 id="oauth1">OAuth1</h3>
     </div>
     <div class="col-md-7 col-sm-12 col-height  docs-code">
       <pre><code class="lang-bash"><span class="hljs-comment"># setting the environment variables on Zapier.com</span>
-$ zapier env 1.0.0 CLIENT_ID 1234
-$ zapier env 1.0.0 CLIENT_SECRET abcd
+$ zapier env:<span class="hljs-built_in">set</span> 1.0.0 CLIENT_ID=1234
+$ zapier env:<span class="hljs-built_in">set</span> 1.0.0 CLIENT_SECRET=abcd
```

Comment:
so it looks like this got some extra special treatment (because of the `set` keyword? or because of the `:`? 🤷 )

but it shows up fine: 

![](https://cdn.zappy.app/aef9faff0e641fc30752ddea6b38e0e2.png)